controllers/helpers.go

```diff
@@ -60,7 +66,7 @@ func doContentStoreRequest(contentStoreClient *contentstore.ContentStoreClient,
 
 	if w != nil {
 		if err != nil {
-			renderer.JSON(w, http.StatusInternalServerError, err)
+			renderer.JSON(w, http.StatusInternalServerError, ErrorResponse{Message: "Unexpected error in request to content-store"})
```

Comment:
This seems to be throwing away some of the error detail.  All errors in Go implement an `Error()` method that returns a string.  Would be good to include this in the response to give some additional context.